packages/stream-metadata/package.json

```diff
@@ -7,9 +7,10 @@
 	"type": "module",
 	"scripts": {
 		"build": "node ./esbuild.config.mjs",
-		"dev": "yarn build && NODE_ENV=development node --experimental-detect-module ./dist/node_esbuild.cjs",
+		"lint": "eslint . --ext .ts",
+		"dev": "yarn build && node --experimental-detect-module ./dist/node_esbuild.cjs",
 		"prettier": "prettier --config ../prettier-config/config.js --ignore-path .prettierignore .",
-		"start": "NODE_ENV=production node --experimental-detect-module ./dist/node_esbuild.cjs"
+		"start": "node --experimental-detect-module ./dist/node_esbuild.cjs"
```

Comment:
@tak-hntlabs can we remove --experimental-detect-module? node should work with .cjs modules by default

Reply:
fixed